hbase-server/src/main/java/org/apache/hadoop/hbase/snapshot/SnapshotDescriptionUtils.java

```diff
@@ -124,6 +124,9 @@ public CompletedSnaphotDirectoriesFilter(FileSystem fs) {
   /** Default value if no start time is specified */
   public static final long NO_SNAPSHOT_START_TIME_SPECIFIED = 0;
 
+  // Default value if no ttl is specified for Snapshot
+  private static final long NO_SNAPSHOT_TTL_SPECIFIED = 0;
+
   public static final String MASTER_SNAPSHOT_TIMEOUT_MILLIS = "hbase.snapshot.master.timeout.millis";
```

Comment:
Actually, here we use ms for snapshot related timeout, is that a good idea to use second for TTL for consistent concern?

Reply:
Good point. However, from the end user perspective, do you think it would be better to provide sec while creating snapshot?
e.g. which one would be preferred for 1 min TTL?

snapshot 't1', 'sp1', {TTL => 60}
snapshot 't1', 'sp1', {TTL => 60000}

multiply by 1000 increases the no for end user to provide with increased TTL

Reply:
So far, from shell perspective,  it is the former. But i'm not sure if the hbase shell convert the 60s to 60000ms.

Reply:
shell doesn't convert it to ms. this will continue to flow as sec only untill this point: https://github.com/apache/hbase/pull/371/files#diff-5fc046382ccaa45c26b89ba3462e8175R92
Please let me know if this looks good to you

Reply:
IIRC, hbase shell accepts seconds like cf's TTL, i prefer `snapshot 't1', 'sp1', {TTL => 60}`. But server side shall be ms, so where the convert happens? Can we follow that convert impl?